td-agent/Rakefile

```diff
@@ -723,6 +728,21 @@ class BuildTask
     cp(src, dest)
   end
 
+  def install_gemfiles
+    ensure_directory(staging_sharedir) do
+      ["Gemfile", "Gemfile.lock"].each do |file|
+        if windows?
+          source_dir = File.join(__dir__, "..", "gemfiles", file == "Gemfile" ? "linux" : "windows")
```

Comment:
Can we unity `windows?` if clause? Or, split for readability?

Reply:
Fixed.